draft-ietf-oauth-selective-disclosure-jwt.md

```diff
@@ -1174,23 +1174,29 @@ Furthermore, re-using the same value to blind a claim name may limit the privacy
 
 ## Unlinkability
 
-Colluding Issuer/Verifier or Verifier/Verifier pairs could link issuance/presentation or two presentation sessions
-to the same user on the basis of unique values encoded in the SD-JWT
-(Issuer signature, salts, digests, etc.). More advanced cryptographic schemes, outside the scope of
-this specification, can be used to prevent this type of linkability.
+Colluding Issuer/Verifier or Verifier/Verifier pairs could link issuance/presentation 
+or two presentation sessions to the same user on the basis of unique values encoded in the SD-JWT
+(Issuer signature, salts, digests, etc.). 
+
+To prevent these types of linkability, various methods, including but not limited to the following ones can be used:
+- Use advanced cryptographic schemes, outside the scope of this specification.
```

Comment:
```suggestion
To prevent these types of linkability, various methods, including but not limited to the following ones can be used:

- Use advanced cryptographic schemes, outside the scope of this specification.
```